src/bootstrap/dist.rs

```diff
@@ -2082,6 +2082,7 @@ impl Step for RustDev {
             "llvm-bcanalyzer",
             "llvm-cov",
             "llvm-dwp",
+            "lld",
```

Comment:
Hmm, I was going to suggest adding `ensure(native::Lld)` somewhere, but I don't see *any* ensure calls in this function. Maybe this step is assuming it's run from some other step? But it seems better to just call `ensure` anyway for all the tools it lists here.

Reply:
I'm not very familiar with this code - I think it would be better to add in `ensure` calls in a separate PR.